pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -214,7 +215,11 @@ private static void start() {
         boolean doLiquibase = cfg.getBoolean(ConfigFile.DO_LIQUIBASE);
         int maxConnections = cfg.getInt(ConfigFile.NUM_POOLED_CONNECTIONS);
 
-        String firecloudKeysLocation = System.getProperty(ConfigFile.FIRECLOUD_KEYS_DIR_ENV_VAR);
+        // For benefit of GAE. Does not like command line options with "=" characters and env variables with "."
+        String firecloudKeysLocation = System.getenv(ConfigFile.FIRECLOUD_KEYS_DIR_ENV_VAR.replace('.', '_'));
```

Comment:
To clarify what I mean in case there's confusion: we don't have any actual production integration with firecloud. So I feel these `firecloud_keys_dir` things are somewhat a relic of past prototyping, and will be nice to remove them so we don't have to add stuff we don't need on the command line.